### DISCUSSION TITLE
Wide Tiles for Apps and Games on Startup

### DISCUSSION THREAD

**Discussion by ChrisElectrical** [valinet/ExplorerPatcher#2259] [Q&A]
Hello everyone! This is my first time posting here so I really appreciate the ones who are taking time out of their day to solve things like this. I have OCD and it has been bothering me that I am so close to my perfect setup, yet so far. I know others understand this feeling too. There is an app called Pepeizq's Tiles For Games that I would like to use. I was hoping that someone can figure out a way to use it! Thank you so much!

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by ChrisElectrical** [reply]
Thank you very much for the response!

**VERSIONS AND BUILDS**
**Windows Version:** 22H2
**OS Build:** 22621.2283
**Explorer Patcher Version:** 22621.2361.58.4

**EXPLORER PATCHER PROPERTIES APPLICATION**
**Taskbar**
**Taskbar Style:** Windows 11 (default)
**Show Search Button:** Yes
**Show Task View Button:** No
**Show Widgets Button:** No
**Automatically Hide The Taskbar:** No
**System Tray**
**Skin Taskbar And Tray Pop-Up Menus:** Yes
**Center Tray Icon Pop-Up Menus:** Yes
**Flyout Behavior For Tray Icon Pop-Up Menus:** Yes
**Show Touch Keyboard Button:** No
**Control Center Button:** Yes
**Show Desktop Button:** Yes
**File Explorer**
All are off except for "Use immersive menus when displaying Windows 10 Context Menus
**Start Menu**
**Start Menu Style:** Windows 10
**Position On Screen:** At screen edge
**Maximum Number Of Frequent Apps To Show:** 6 (default)
**When Using Multiple Displays, Open Start On This Monitor When Invoked Using The Keyboard:** Primary monitory (default)
**Show More Tiles:** Yes
**Corner Preference:** Not rounded
**Display Mode:** Default
**App List:** Hide
**Window Switcher (Alt+Tab) style:** Windows 11 (default)
**Other**
All are off
**Updates**
Not recommended settings are off
**Advanced**
All are off except for "Enable Symbols Download" which is a yes.

**Reply by pyrates999** [reply]
Is this what you're referring to? https://github.com/pepeizq/Tiles-Games

**Reply by ChrisElectrical** [reply]
Yep! That's the one!

**Reply by pyrates999** [reply]
Then you should post on their help forum to get help to use it.  This isn't an issue for EP.

**Reply by ChrisElectrical** [reply]
Yeah, that was the original plan. The guy doesn't have any place to submit such a thing, unfortunately. I looked at his website, but I don't see any indication of him being active on the contact page. So, I was hoping someone would be able to figure out how to make it happen here. I did however find a comment that addressed this but I am not sure if it will cause any issues.

https://github.com/valinet/ExplorerPatcher/issues/1534#issuecomment-1601017536

Please let me know if you think this would cause any problems.

**Reply by Poopooracoocoo** [reply]
do you know how pepeizq's Tiles for Games is different to tile iconifier?

**Comment by ChrisElectrical** [comment]
> do you know how pepeizq's Tiles for Games is different to tile iconifier?

Unfortunately, I have never heard of Tile Iconifier till now. So I do not. What I do know is that Pepeizq's Tiles for Games was updated this year I believe and Tile Iconifier was last updated in 2021.
